Tidying up the Interacting section (#529)
Unity-Technologies/com.unity.probuilder: Documentation~/Editor_Windows.md

# Editor windows

![Editor windows](images/pb_editors.png)

> Editor windows provide [features](tool-panels.md) with extended functionality. For example, the UV Editor window (in the example image above) allows you to perform advanced texture manipulations, including texture mapping, UV unwrapping, and tiling.

Reviewer: ```suggestion
Editor windows provide [features](tool-panels.md) with extended functionality. For example, you can use the UV Editor window to perform advanced texture manipulations, including texture mapping, UV unwrapping, and tiling.
```
